lib/helpers.js

```diff
@@ -210,7 +211,8 @@ var convertValue = module.exports.convertValue = function (schema, options, valu
   // Attempt to parse the string as JSON if the type is array or object
   if (['array', 'object'].indexOf(type) > -1) {
```

Comment:
I think we should only be doing this if the value is a string **and** the requested type is `array` or `object`.  While not completely related, can you change this line to the following:

```js
if (['array', 'object'].indexOf(type) > -1 && _.isString(value)) {
```